.yarn/patches/dpdm-npm-3.12.0-0dfdd8e3b8.patch

```diff
@@ -0,0 +1,42 @@
+diff --git a/lib/parser.js b/lib/parser.js
+index 43e4aff85acc56f5360f80b639062157e64b61c1..23d3e7b37004f6b8787d14cbe4704bc626ac526f 100644
+--- a/lib/parser.js
++++ b/lib/parser.js
+@@ -46,8 +46,7 @@ function parseTreeRecursive(context, request, options, output, resolve) {
+                 node.expression.kind === typescript_1.default.SyntaxKind.ImportKeyword &&
+                 node.arguments.length === 1 &&
+                 typescript_1.default.isStringLiteral(node.arguments[0])) {
+-                newRequest = node.arguments[0].text;
+-                kind = consts_1.DependencyKind.DynamicImport;
++                return;
```

Comment:
This library by default considers dynamic imports to be a circular dependency, which we don't want